src/crimson/osd/osd_operations/client_request.cc

```diff
@@ -203,12 +203,9 @@ ClientRequest::interruptible_future<>
 ClientRequest::process_pg_op(
   Ref<PG> pg)
 {
-  return pg->do_pg_ops(
-    m
-  ).then_interruptible([this, pg=std::move(pg)](MURef<MOSDOpReply> reply) {
-    // TODO: gate the crosscore sending
```

Comment:
Is this comment no loner relevant?

Reply:
I'm fine with removing it -- the method call itself is enough.

Reply:
I suppose this comment is still valid -- crosscore sending isn't chained by `send_with_throttling()` as the future here is only for throttling.

A connection doesn't implement a per-shard gate -- I suppose it would be simpler to reuse the OSD per-shard gate to make sure graceful shutdown will wait for crosscore sending properly.

Reply:
I can re-add it.

Reply:
readded